packages/11ty/_plugins/globalData/index.js

```diff
@@ -73,15 +101,27 @@ module.exports = function(eleventyConfig, directoryConfig) {
   const parse = parser(eleventyConfig)
 
   for (const file of files) {
+
     const { name: key } = path.parse(file)
     let value = parse(path.join(dir, file))
-    if (key === 'publication') {
+    // console.log(key,value)
+
+    switch (key) {
+    case 'config':
+      value = validateConfig(value)
+      break
+    case 'publication':
       value = validatePublication(value)
+      break
```

Comment:
Indent `case` blocks and add a `default` block
```suggestion
      case 'config':
        value = validateConfig(value)
        break
      case 'publication':
        value = validatePublication(value)
        break
      default:
        break
```
however we might refactor `validate` to accept a `type` and a `value`, which would allow the above to be a sinlge statement, for example:
```
// @example validate('publication', data)
value = validate(key, value)
```

Reply:
Erm, *this* is done -- cleans that up considerably and makes what I was trying to do with the `switch` statements much clearer.